service/README.md

```diff
@@ -0,0 +1,157 @@
+# nydus-service
```

Comment:
Could we put a link to reference it in [README.md](https://github.com/dragonflyoss/image-service/blob/master/README.md)?

Reply:
What about adding a part named "Reuse Nydus Services" in [README.md](https://github.com/dragonflyoss/image-service/blob/master/README.md) to put this link.

Reply:
Good, and give a brief description.